Makefile

```diff
@@ -77,11 +81,19 @@ clean:
 	cargo clean
 	go clean ./...
 
+# DEPRECATED - please use build-stellar-rpc instead
 # the build-soroban-rpc build target is an optimized build target used by
-# https://github.com/stellar/pipelines/stellar-horizon/Jenkinsfile-soroban-rpc-package-builder
+# https://github.com/stellar/pipelines/blob/master/soroban-rpc/Jenkinsfile-soroban-rpc-package-builder
 # as part of the package building.
 build-soroban-rpc: build-libs
-	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o soroban-rpc -trimpath -v ./cmd/soroban-rpc
+	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o ${SOROBAN_RPC_BINARY} -trimpath -v ./cmd/soroban-rpc
+
+# the build-stellar-rpc build target is an optimized build target used by
+# https://github.com/stellar/pipelines/blob/master/soroban-rpc/Jenkinsfile-soroban-rpc-package-builder
+# as part of the package building.
+build-stellar-rpc: build-libs
+	go build -ldflags="${GOLDFLAGS}" ${MACOS_MIN_VER} -o ${STELLAR_RPC_BINARY} -trimpath -v ./cmd/soroban-rpc
+
```

Comment:
All these changes seem unrelated to the PR

Reply:
landed those changes to `main` yesterday, that's why it is showing up in diff !! `v22-breaking-change` need to pull new changes from main

Reply:
@2opremio can you merge `main` to `v22-breaking-change` so that I can close this PR 😄

Reply:
^^ This is fixed now !!